.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -200,18 +207,58 @@ jobs:
       - name: Fail if no images have been built
         run: if [ $(wc -l < ${{ matrix.distro }}-container-images) -le 1 ]; then exit 1; fi
 
-      - name: Upload container images artifact
+      - name: Scan built container images
+        run: src/kayobe-config/tools/scan-images.sh ${{ matrix.distro }} ${{ needs.generate-tag.outputs.kolla_tag }}
+
+      - name: Upload Trivy scan results artifact
```

Comment:
Should we just chuck everything in one artifact (per-distro)?

Reply:
Done